go/test/endtoend/cluster/cluster_util.go

```diff
@@ -265,7 +265,7 @@ func WaitForReplicationPos(t *testing.T, tabletA *Vttablet, tabletB *Vttablet, v
 		msg := fmt.Sprintf("%s's replication position to catch up to %s's;currently at: %s, waiting to catch up to: %s", tabletB.Alias, tabletA.Alias, replicationPosB, replicationPosA)
 		select {
 		case <-ctx.Done():
-			assert.FailNowf(t, "Timeout waiting for condition '%s'", msg)
+			assert.FailNow(t, "Timeout waiting for condition '%s'", msg)
```

Comment:
This was not printing the right thing before and also ran these tests to verify the utils change.